torch/autograd/gradcheck.py

```diff
@@ -545,6 +613,7 @@ def gradgradcheck(
             the second derivative.
         check_undefined_grad (bool, options): if True, check if undefined output grads
             are supported and treated as zeros
+        check_forward(bool, options): if True, check the forward mode AD gradient
```

Comment:
options -> optional

Reply:
Copy pasted the options from the one above... But I think you fixed this one in master? I'll update!

Reply:
My PR got reverted haha. I'll re-submit it later today